pytimber/pytimber.py

```diff
@@ -68,8 +68,19 @@ class LoggingDB(object):
     except NameError:
         print('ERROR: jpype is note defined!')
 
+    def _read_conf_file(self, conf_filename):
```

Comment:
This might be a good use of ConfigParser:

https://docs.python.org/3/library/configparser.html

Reply:
Is it python2 compatible? Do we need to print the warning? The default settings are the best settings for anybody, besides the few that have privileged access. The wording of the warning does not suggest that.

Reply:
ConfigParser is compatible with Python 2, although you have to take care of the name ("ConfigParser" in Python 2 compared to "configparser" in Python 3). But that's easy to work around.

I am not really sure of the point of this patch though. As you say, few people have their own app/client IDs. If they do, they can already pass them as parameters to their scripts. So why also have the option for a configuration file?

Reply:
@levens i know configparser but if we want to be able to use the old configuration.properties format used with the official CLI we need a new parser and i see no point for that.
@rdemaria the warning seems to reflect what we discussed in #58 but sure, we can change the message.

Reply:
@levens i suggested this patch after noticing very poor performance using the default credentials (i have jobs with a default timeout of 1s, which should be more than enough for the last value of a single variable, but they kept failing with the appid/clientid). in the future it will also be possible to read the other settings from the configuration.properties, which could be useful

Reply:
Perhaps you can put the message the docstring and/or change the warning into debug or info.